etc/testing/travis_cache.sh

```diff
@@ -11,6 +11,7 @@ set -ex
 dirs=(
   "${HOME}/.cache"
   "${HOME}/cached-deps"
+  "${HOME}/gopath/pkg/mod"
```

Comment:
Similar to above: would it make sense to use `$GOPATH` instead of `$HOME/gopath`?